.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+package-lock.json
```

Comment:
this one should be in a separated PR.

Reply:
https://github.com/pinojs/pino-caller/pull/19